ios/engine/KMEI/KeymanEngine/Classes/kmp.json/KMPMetadata.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  KMPInfo.swift
+//  KeymanEngine
+//
+//  Created by Joshua Horton on 6/2/20.
+//  Copyright © 2020 SIL International. All rights reserved.
+//
+
+import Foundation
+
+public class KMPMetadata: Codable {
```

Comment:
It occurs to me that we should include urls for:

1. schema definition to which this conforms (from api.keyman.com/schemas), applicable version etc.
2. documentation for the schema (from help.keyman.com/developer)

And this should be in all the places where we have data definitions. This is inconsistent in the code base but better to start somewhere, now, ...